Makefile

```diff
@@ -96,6 +96,7 @@ release-announce: .release-functest
 
 cluster-up:
 	./cluster-up/up.sh
+	./hack/install-cnao.sh
```

Comment:
Images should be pre-pulled in kubevirtci provider. In kubevirtci/cluster-up, there should be a flag enabling installation of these manifests. That way we get stable snapshot of our released components that would be available on demand. With kubevirt we would install them, with cnao, they would stay dormant.